config/client.yaml

```diff
@@ -3,9 +3,6 @@ config:
     release-v1.15:
       konflux:
         enabled: true
-      imageOverrides:
-      - name: CLI_ARTIFACTS
-        pullSpec: brew.registry.redhat.io/rh-osbs/openshift-serverless-1-kn-cli-artifacts-rhel8:1.15.0
```

Comment:
@dsimansk this is being deleted

Reply:
/hold for now

Reply:
Related to https://github.com/openshift-knative/hack/pull/459#discussion_r1862320348

Reply:
IMO, fine to unhold and proceed. Per your other comment, It's not used per this config anyway.

Reply:
I think this is solved now (https://github.com/openshift-knative/hack/pull/479)
/unhold